docs/docs/05-migrations/13-v8-to-v9.md

```diff
@@ -157,31 +230,40 @@ func AssertEvents(
 )
 ```
 
-- The `QueryServer` interface has been removed from the `TestChain` struct. Submodule query servers can be constructed directly by passing their associated keeper to the appropriate constructor function. For example:
+- The [`QueryServer` interface has been removed from the `TestChain` struc](https://github.com/cosmos/ibc-go/blob/v8.0.0/testing/chain.go#L61). Submodule query servers can be constructed directly by passing their associated keeper to the appropriate constructor function. For example:
 
 ```golang
 clientQueryServer := clientkeeper.NewQueryServer(app.IBCKeeper.ClientKeeper)
 ```
 
-#### API deprecation notice
+### API deprecation notice
 
-The testing package functions `coordinator.Setup`, `coordinator.SetupClients`, `coordinator.SetupConnections`, `coordinator.CreateConnections`, and `coordinator.CreateChannels` have been deprecated and will be removed in v10.
-Please use the new functions `path.Setup`, `path.SetupClients`, `path.SetupConnections`, `path.CreateConnections`, `path.CreateChannels`.
+- The testing package functions `Setup`, `SetupClients`, `SetupConnections`, `CreateConnections`, and `CreateChannels` of the `Coordinator` type have been deprecated and will be removed in v10. Please use the new functions `Setup`, `SetupClients`, `SetupConnections`, `CreateConnections`, `CreateChannels` of the `Path` type.
 
 ## Relayers
 
-- Renaming of event attribute keys in [#5603](https://github.com/cosmos/ibc-go/pull/5603).
-- Removal of duplicate non-hexlified event attributes in [#6023](https://github.com/cosmos/ibc-go/pull/6023).
+### Events
 
-## IBC Light Clients
+#### 02-client
 
-### API removals
+- The [`header` attribute](https://github.com/cosmos/ibc-go/blob/v8.0.0/modules/core/02-client/keeper/events.go#L60) has been removed from the `update_client` event in [\#5110](https://github.com/cosmos/ibc-go/pull/5110).
```

Comment:
This change was discussed a while back with hermes team and they were fine with it, but I will bring it up again to make sure they will be ready.